apps/graphql/src/services/fetch/helpers/extendHttpHeaders.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+import type { Options } from '../globalFetch';
+
+export default function extendHttpHeaders(
```

Comment:
Name of this function was slightly confusing for me at first with parameters combination.
And I was thinking why it is not called for example something like `mergeFetchOptions`.

But I understand that we need it just for headers merge and there will not be beccesity to extend it also for other options merging.
So it is quite fine like this.